hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -202,6 +202,30 @@ static inline void vlapic_build_x2apic_id(struct acrn_vlapic *vlapic)
 	lapic->ldr.v = (cluster_id << 16U) | (1U << logical_id);
 }
 
+static inline uint32_t vlapic_find_isrv(const struct acrn_vlapic *vlapic)
```

Comment:
find_highest_isrv, not isrv

Reply:
SDM, Vol 3, Chap 10.6.2.4 Lowest Priority Delivery Mode: "the IRRV value is the vector number
for the highest priority bit that is set in the IRR (see Figure 10-20) or 00H (if no IRR bit is set), and the ISRV value is the vector number for the highest priority bit that is set in the ISR (see Figure 10-20). ". So I think isrv is enough ?

Reply:
what is full name of isrv?  Make sure the meaning is readable and accurate in meaning.

Reply:
just upercase "ISRV".